src/common/SamplePlayer.js

```diff
@@ -14,6 +14,10 @@ export default class SamplePlayer {
     return this._pads
   }
 
+  getMasterVolume() {
+    return this.samplePlayers.volume.value
+  }
+
```

Comment:
How come you work with class based components here? Just curious to know.

Reply:
Oh, it is not a class based componend, SamplePlayer is a class itself outside of react. ;-)